.github/workflows/qiita-plugin-ci.yml

```diff
@@ -91,10 +93,10 @@ jobs:
           minimap2_version=`minimap2 --version`
           fastp_version=$(fastp --version 2>&1)
           if [[ $samtools_version != *"1.11"* ]]; then echo "wrong samtools version", $samtools_version; exit 1; fi
-          if [[ $minimap2_version != *"2.17"* ]]; then echo "wrong minimap2 version", $minimap2_version; exit 1; fi
-          if [[ $fastp_version != *"20.1"* ]]; then echo "wrong fastp version", $fastp_version; exit 1; fi
+          if [[ $minimap2_version != *"2.24"* ]]; then echo "wrong minimap2 version", $minimap2_version; exit 1; fi
```

Comment:
It may be good to check with Cameron if this matters. Is the version different to what's in upstream processing?

Reply:
After review, we decided to pin the minimap2 version and update the fastp one ...